go.mod

```diff
@@ -164,11 +164,11 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apiextensions-apiserver v0.28.0 // indirect
```

Comment:
It seems `k8s.io/apiextensions-apiserver` was not updated earlier, but it has been updated in the `replace` section. Please update the version here too to prevent confusion.